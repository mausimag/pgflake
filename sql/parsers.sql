SET pgflake.instance_id TO 5;

CREATE EXTENSION IF NOT EXISTS pgflake;

CREATE TABLE simple_test (x bigint, y bigint NOT NULL DEFAULT pgflake_generate());

INSERT INTO simple_test (x) SELECT generate_series(1, 10);

SELECT COUNT(1) FROM simple_test WHERE pgflake_extract_instance(y) = 5;

SELECT pgflake_extract_sequence(y) FROM simple_test LIMIT 1;

DROP TABLE simple_test;

DROP EXTENSION pgflake;
