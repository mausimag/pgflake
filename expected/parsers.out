SET pgflake.instance_id TO 5;
CREATE EXTENSION IF NOT EXISTS pgflake;
CREATE TABLE simple_test (x bigint, y bigint NOT NULL DEFAULT pgflake_generate());
INSERT INTO simple_test (x) SELECT generate_series(1, 10);
SELECT COUNT(1) FROM simple_test WHERE pgflake_extract_instance(y) = 5;
 count 
-------
    10
(1 row)

SELECT pgflake_extract_sequence(y) FROM simple_test LIMIT 1;
 pgflake_extract_sequence 
--------------------------
                        0
(1 row)

DROP TABLE simple_test;
DROP EXTENSION pgflake;
